site/examples/mentions.js

```diff
@@ -83,10 +83,14 @@ const MentionExample = () => {
         if (selection && Range.isCollapsed(selection)) {
           const [start] = Range.edges(selection)
           const wordBefore = Editor.before(editor, start, { unit: 'word' })
-          const before = Editor.before(editor, wordBefore)
-          const beforeRange = Editor.range(editor, before, start)
-          const beforeText = Editor.text(editor, beforeRange)
-          const beforeMatch = beforeText.match(/^@(\w+)$/)
+          let before
+          if (wordBefore) before = Editor.before(editor, wordBefore)
```

Comment:
Instead of using the `if` statements, could you use `&&` like:

```js
const before = wordBefore && ...
```

Reply:
Thanks for the reminder!